src/Meziantou.Framework.NuGetPackageValidation/Internal/SharedHttpClient.cs

```diff
@@ -46,7 +59,7 @@ protected override async Task<HttpResponseMessage> SendAsync(HttpRequestMessage
                         throw;
                 }
 
-                await Task.Delay(100 * i, cancellationToken).ConfigureAwait(false);
+                await Task.Delay(delay is { } someDelay && someDelay > TimeSpan.Zero ? someDelay : TimeSpan.FromMicroseconds(100 * i), cancellationToken).ConfigureAwait(false);
```

Comment:
I think you are right, the default delay is too aggressive. I think we can start with `TimeSpan.FromMilliseconds(200 * i)`. You can implement something smarter if you prefer.

Reply:
> I think we can start with `TimeSpan.FromMilliseconds(200 * i)`.

Done with e01f7d45af4cfd8b2e3fe683e3fbb13b572bdbc8.

Reply:
> You can implement something smarter if you prefer.

I implemented a more exponential strategy with 692eacbc4929a835efab87ca12497967e2ebe17d. I can revert if you object. The [GitHub documentation recommends something similar](https://docs.github.com/en/rest/overview/resources-in-the-rest-api?apiVersion=2022-11-28#secondary-rate-limits):

> Otherwise, wait for an exponentially increasing amount of time between retries, and throw an error after a specific number of retries.